husky_components/dropzone/main.js

```diff
@@ -284,22 +284,26 @@ define([], function() {
                         this.addFiles(event.originalEvent.dataTransfer.files);
                     }
                 }.bind(this));
+                this.$dropzone.on('click', function() {
+                    if (!!this.options.cancelUploadOnOverlayClose) {
```

Comment:
Why have you moved this one out of the `closeOverlay` function? Is there any situation in where reopening the dropzone should not drop all the files?